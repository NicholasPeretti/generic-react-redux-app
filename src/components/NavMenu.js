import styled from 'styled-components'

const NavMenu = styled.ul`
  display: flex;
  flex-direction: row;
  list-style-type: none;
`

export default NavMenu
