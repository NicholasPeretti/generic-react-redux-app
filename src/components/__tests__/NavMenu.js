import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import NavMenu from '../NavMenu'

describe('NavMenu', () => {
  PresentationalTest(NavMenu)

  test('Should match snapshot', () => {
    const wrapper = shallow(<NavMenu/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
