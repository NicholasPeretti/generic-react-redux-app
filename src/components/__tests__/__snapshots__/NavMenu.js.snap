// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMenu Should match snapshot 1`] = `"<ul class=\\"sc-bdVaJa hoBDDC\\"></ul>"`;
